### PR TITLE
fix:  Removing currency symbol filter

### LINKED ISSUE #6428: Blank Currency Symbol in `Orders` PDF file generated
**Describe the bug**

Blank Currency Symbol in `Orders` PDF file generated

**To Reproduce**

1. Go to 'Events/Tickets/Order'
2. Click on 'Export As PDF'
3. See error

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #6428 

#### Short description of what this resolves:

Removes the currency symbol template filter which was not able to render most of the currency symbols due to a rendering issue with PISA

### CODE REVIEW

**Comment by kushthedude** [comment]
@CosmicCoder96 @iamareebjamal Please Review

File: app/templates/pdf/attendees_pdf.html [hunk 1]
```diff
@@ -123,7 +123,7 @@ <h3 style="text-align:center;">{{ holders[0].event.name }} - {{ ("Attendees List
                         {{ holder.email }}<br>
                     </strong>
                     {% if holder.ticket.price %}
-                        {{ holder.event.payment_currency | currency_symbol }}{{ holder.ticket.price | money }}
+                        {{ holder.event.payment_currency }}{{ holder.ticket.price | money }}
```
Comment (shreyanshdwivedi) [inline app/templates/pdf/attendees_pdf.html]: @kushthedude please add screenshot of the change for reference.

**Comment by shreyanshdwivedi** [comment]
@kushthedude can you please update this?

File: app/templates/pdf/attendees_pdf.html (hunk 1 above)
Comment (kushthedude) [reply app/templates/pdf/attendees_pdf.html]: ## Problem
Screenshot 2019-09-02 at 7 49 57 AM

## Now
Screenshot 2019-09-02 at 7 49 47 AM

**Comment by kushthedude** [comment]
> @kushthedude can you please update this?

pr was already ready, added the ss

**Comment by kushthedude** [comment]
@uds5501 Merge

**Pull request merged**